zmon_cli/client.py

```diff
@@ -692,6 +692,36 @@ def get_alert_data(self, alert_id: int, **kwargs) -> dict:
 
         return self.json(resp)
 
+    @trace(pass_span=True)
+    @logged
+    def get_alert_status(self, alert_id: int, **kwargs) -> dict:
+        """
+        Retrieve alert Status.
+
+        Response is a ``dict`` with entity ID as a key, and check return value as a value.
+
+        :param alert_id: ZMON alert ID.
+        :type alert_id: int
+
+        :return: Alert data dict.
+        :rtype: dict
+
+        Example:
+
+        .. code-block:: json
```

Comment:
I think this example response is not accurate. It is slightly different from the call with ``all-entities``